gui/translationhandler.cpp

```diff
@@ -30,15 +30,15 @@
 static void unused()
 {
 #if ((QT_VERSION >= 0x040000)&&(QT_VERSION < 0x050000))
-    Q_UNUSED(QT_TRANSLATE_NOOP("QDialogButtonBox", "OK"));
```

Comment:
Imho, stylistically I prefer to keep these redundant semicolons. As a Cppcheck developer I don't like that macros contain semicolons.

Reply:
Undone and with annotation about retention.